app/models/concerns/auditable_model.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module AuditableModel
```

Comment:
Why call it this ? It already lives under the `models/` directory
Just call it `auditable`

Reply:
we already have auditable under controllers, its confusing to call it the same?